code/game/atoms.dm

```diff
@@ -290,6 +290,7 @@
 				to_chat(user, "[total_volume] units of various reagents")
 		else
 			to_chat(user, "Nothing.")
+	SendSignal(COMSIG_PARENT_EXAMINE, user)

```

Comment:
Yeah, it would be determined by the order that components are added. I don't really have any ideas for how to control this...